index.html

```diff
@@ -2734,21 +2750,21 @@ <h4>Pagination with jump to page</h4>
         <article id="popovers">
             <h2>Popovers @TODO</h2>
 
-            <button type="button" class="btn btn-primary" data-container="body" data-toggle="popover" data-placement="left" title="Dismissible popover" data-content="Vivamus sagittis lacus vel augue laoreet rutrum faucibus.">
+            <button type="button" class="btn btn-primary" data-container="body" data-toggle="popover" data-trigger="hover" data-placement="left" title="Dismissible popover" data-content="Vivamus sagittis lacus vel augue laoreet rutrum faucibus.">
                 Popover on left (with title)
             </button>
-            <button type="button" class="btn btn-outline-primary" data-container="body" data-toggle="popover" data-placement="top" data-content="Vivamus sagittis lacus vel augue laoreet rutrum faucibus.">
+            <button type="button" class="btn btn-outline-primary" data-container="body" data-toggle="popover" data-trigger="hover"  data-placement="top" data-content="Vivamus sagittis lacus vel augue laoreet rutrum faucibus.">
```

Comment:
```suggestion
            <button type="button" class="btn btn-outline-primary" data-container="body" data-toggle="popover" data-trigger="hover" data-placement="top" data-content="Vivamus sagittis lacus vel augue laoreet rutrum faucibus.">
```
is it possible to improve indentation, for a better review?